README.md

```diff
@@ -52,7 +52,69 @@ struct GetCatImages: Request {
 }
 ```
 
-#### Make your request and handle the result:
+### Make your request using `async`/`await` and handle the result:
+
+```swift
+Task {
+    do {
+        let catImages = try await netable.request(GetCatImages())
+        if let firstCat = catImages.first,
+           let url = URL(string: firstCat.url),
+           let imageData = try? Data(contentsOf: url) {
+            self.catsImageView1.image = UIImage(data: imageData)
+        }
+
+        if let lastCat = catImages.last,
+           let url = URL(string: lastCat.url),
+           let imageData = try? Data(contentsOf: url) {
+            self.catsImageView2.image = UIImage(data: imageData)
+        }
+    } catch {
+        let alert = UIAlertController(
+            title: "Uh oh!",
+            message: "Get cats request failed with error: \(error)",
+            preferredStyle: .alert
+        )
+
+        alert.addAction(UIAlertAction(title: "OK", style: .cancel))
+        self.present(alert, animated: true, completion: nil)
+    }
+}
+```
+
+### Making a request with Combine
+
+```swift
+netable.request(GetCatImages())
+    .receive(on: RunLoop.main)
```

Comment:
It would be nice if the emission happend on the main thread so callers didn't have to explicitly receive on the main thread